### ISSUE TITLE
Add capability of putting in more than one category to a recipe

### ISSUE THREAD

**Issue by kylec32** [kylec32/recipe-holder#2]
Currently if you try to put in more than one category it doesn't handle that. Extend the functionality so that it understands that a comma separates different categories would be great.

**Comment by jsjaskaran** [comment]
Hey @kylec32. I would love to help you on this one. Can you explain a little bit more as to how you want the whole functionality to work or look? Do you want them to look like tags?

**Comment by kylec32** [comment]
Yeah the bootstrap tags is actually exactly how I saw it in mind (although I'd isn't my strong suit so I figured I would leave it up to the implementor but it's great to hear the idea validated). As for the rest of the functionality it just needs to work with the filtering on the recipe list page. I can't remember if currently it allows you to select more than one category but if it doesn't than that would need to be adjusted and then make it so the back end with return the recipes that are in the categories (or the selected filter categories are a subset of the recipes categories). Hopefully that all makes sense. 

I really appreciate the help, let me know if you have any other questions or ideas.

**Comment by jsjaskaran** [comment]
Sorry for the late reply, I've been stuck at work for the past couple of days. Anyway I just created a pull request with feature you requested.

**Comment by kylec32** [comment]
Awesome! Thank you so much. I'll take a look soon!
